Usiqee/Module/Auth/ForgetPasswordVC.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ForgetPasswordVC.swift
+//  Usiqee
+//
+//  Created by Amine on 02/05/2021.
+//
+
+import UIKit
+import Firebase
+
+protocol ForgetPasswordVCDelegate: class {
+    func didReset()
+}
+
+class ForgetPasswordVC: UIViewController {
+
+    // MARK: - IBOutlet
+    @IBOutlet weak private var email: MTextField!
+    @IBOutlet weak private var validButton: UIButton!
+    
+    // MARK: - Variables
+    weak var delegate: ForgetPasswordVCDelegate?
+    
+    // MARK: - LifeCycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpView()
+    }
+    
+    // MARK: - Privates
+    private func setUpView() {
+        setupEmailTextField()
+        setupSignInButton()
+    }
+    
+    private func setupEmailTextField() {
+        email.delegate = self
+        email.placeHolder = L10N.user.form.mail
+        email.textContentType = .emailAddress
+        email.returnKeyType = .send
+    }
+    
+    private func setupSignInButton() {
+        validButton.setTitle(L10N.ForgetPassword.form.valid, for: .normal)
+        validButton.layer.cornerRadius = 20
+        validButton.clipsToBounds = true
+        validButton.setBackgroundColor(Colors.purple)
```

Comment:
Look like this button configuration have been repeat in `NotLoggedView.swift`, can we have a global button for that ?

Reply:
indeed